app/assets/javascripts/host_edit.js

```diff
@@ -165,6 +165,16 @@ function update_capabilities(capabilities) {
 
 var stop_pooling;
 
+if (window.location.href.contains('hostgroup')) {
+  document.addEventListener('DOMContentLoaded', function() {
```

Comment:
Console error on this: `TypeError: window.location.href.contains is not a function`